setenv_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package utils_test
+import (
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/utils"
+)
+
+type SetenvSuite struct{}
+
+var _ = gc.Suite(&SetenvSuite{})
+
+var setenvTests = []struct {
```

Comment:
It seems odd to have these as a slice outside of the function given that they depend so concretely on the initial setup.